src/utils/beacon/BeaconPeers.test.tsx

```diff
@@ -88,6 +88,34 @@ describe("<BeaconPeers />", () => {
       }
     });
 
+    it("resolves duplicating name for different peer connections", async () => {
```

Comment:
Imo the test is redundant. Have you seen it fail?

Reply:
Yes, with the previous key `key={peerInfo.name}``

I can remove the test if you think we don't need it